cost/cloud_cost_anomaly_alerts/README.md

```diff
@@ -21,8 +21,10 @@ Any point outside of the bands is considered as anomalous. If multiple cost anom
 - *Minimum Period Spend* - minimum daily spend needed to record an anomaly as an incident.
   - Minimum Value: 0
   - Default: 1000
-- *Cost Anomaly Dimensions* - Dimensions to group data for the analysis. Multiple dimensions can be provided. Currently, custom tags cannot be used as dimensions.
+- *Cost Anomaly Dimensions* - Dimensions to group data for the analysis. Multiple dimensions can be provided.
   - Default: ["Cloud Vendor Account", "Cloud Vendor", "Service"]
+  - Id(s) or name(s) for Rule Based Dimensions to group data for the analysis. Multiple rbd id(s) or name(s) can be provided. This allows for better visualization of data that was not tagged or difficult to tag in the native cloud platform, these new dimensions in Optima will be populated according to rules you define, you can use the new dimensions to look at your cost from new meaningful perspectives, a common use case is to create several dimensions like 'CCO Environments' or 'Product Line', populated based on know account values.
+  - Filters can now be added to the same values ​​of the dimensions, you can filter 1 to 1 value per dimension e.g. Service=AmazonEC2 , Cloud Vendor=AWS , Cloud Vendor Account Name=Flexera-prod.
```

Comment:
Can we change to 

ID(s) or name(s) for Rule Based Dimensions to group data for analysis. Multiple RBD ID(s) or name(s) can be provided. This allows for better visualization of data that was not tagged or difficult to tag in the native cloud platform, these new dimensions in Optima will be populated according to rules you define, you can use the new dimensions to look at your cost from new meaningful perspectives, a common use case is to create several dimensions like 'CCO Environments' or 'Product Line', populated based on known account values. - Filters can now be added to the same values ​​of the dimensions. You can filter 1 to 1 value per dimension e.g. Service=AmazonEC2 , Cloud Vendor=AWS , Cloud Vendor Account Name=Flexera-prod.